src/Symfony/Component/Notifier/Message/EmailMessage.php

```diff
@@ -35,7 +35,7 @@ public function __construct(RawMessage $message, Envelope $envelope = null)
         $this->envelope = $envelope;
     }
 
-    public static function fromNotification(Notification $notification, Recipient $recipient, string $transport = null): self
```

Comment:
Alternative: for the $transport, we could set the transport directly from this method then? But the current code uses the setter everytime.